bin/check-memory.sh

```diff
@@ -46,7 +46,7 @@ if [[ -f /etc/redhat-version && `awk '{print $3}' /etc/redhat-release` =~ ^2[0-9
    redhat_version=1
 elif [[ -f /etc/redhat-version && `awk '{print $7}' /etc/redhat-version` =~ ^7\.[0-9]{1} ]]; then
    redhat_version=1
-elif [[ -f /etc/redhat-release && `awk '{print $4}' /etc/redhat-release` =~ ^7\.[0-9]{1} ]]; then
+elif [[ -f /etc/redhat-release && `awk '{print $7}' /etc/redhat-release` =~ ^7\.[0-9]{1} ]]; then
```

Comment:
this breaks centos, which has the version in $4. I suspect the elif above that was supposed to say /etc/redhat-release instead of /etc/redhat-version. Can you fix that so this works on both Centos and RHEL.